tests/plugins/tabletools/cellproperties.js

```diff
@@ -67,20 +67,6 @@
 
 		// #11439
 		'test load and update field values (#5)': function() {
-			this.doTest( 'table-6', function( dialog ) {
-				assert.areSame( '', dialog.getValueOf( 'info', 'height' ) );
-				assert.areSame( 'px', dialog.getValueOf( 'info', 'widthType' ) );
-				assert.areSame( '', dialog.getValueOf( 'info', 'wordWrap' ) );
-				assert.areSame( '', dialog.getValueOf( 'info', 'rowSpan' ) );
-
-				dialog.setValueOf( 'info', 'width', 100 );
```

Comment:
should we keep the stuff that doesn't have to do with the background color around still?

Reply:
Correct. I put them back. Just omitted bgColor part